realm/realm-library/src/main/java/io/realm/internal/Collection.java

```diff
@@ -594,6 +711,16 @@ private static native long nativeCreateResultsFromLinkView(long sharedRealmNativ
 
     private static native void nativeDelete(long nativePtr, long index);
 
+    private static native void nativeSetNull(long nativePtr, long columnIndex);
+    private static native void nativeSetBoolean(long nativePtr, long columnIndex, boolean value);
+    private static native void nativeSetInt(long nativePtr, long columnIndex, long value);
```

Comment:
this should be name `nativeSetLong` (since core uses long to represent Int, long ...) similar to [setLong](https://github.com/realm/realm-java/blob/19e45278726ecb90f1d37b39146753b46537c4bb/realm/realm-library/src/main/java/io/realm/internal/Table.java#L518)